.vs/config/applicationhost.config.orig

```diff
@@ -0,0 +1,1047 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file. Every file ending with .orig is a temporary file used for solving merge conflicts